source/FunkinLua.hx

```diff
@@ -1388,6 +1388,11 @@ class FunkinLua {
 			return Std.parseInt(color);
 		});
 
+		Lua_helper.add_callback(lua, "getColorFromRgb", function(r:Int, g:Int, b:Int) {
+			var color = FlxColor.fromRGB(r, g, b);
+			return color;
```

Comment:
this would return a FlxColor object
`return FlxColor.toHexString(color, false);` would probably work instead